### PR TITLE
Add Maven Topic

### PR DESCRIPTION
**Pull request by RohanNagar**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

***********EDITING AN EXISTING TOPIC OR COLLECTION************

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:
Maven is a popular tool among Java developers and is often searched for. Having a description of this topic as links to more documentation will help developers searching for it on GitHub for the first time.

************CURATING A NEW COLLECTION************

- [ ] I've formatted my changes as a new folder directory, named for the collection as it appears in the URL on GitHub (e.g. `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this collection should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by RohanNagar** [comment]
Can someone help suggest what the Maven logo should look like? It seems like 288 x 288 is expected, but I don't know of any Maven square logo.

File: topics/maven/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+aliases: apache-maven
+created_by: Apache Software Foundation
+display_name: "Maven"
```
Comment (MikeMcQuaid) [inline topics/maven/index.md]: Does this need the quotes?

File: topics/maven/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+github_url: https://github.com/apache/maven
+logo: maven.png
+released: July 13, 2004
+short_description: "Maven is a build automation tool used primarily for Java projects."
```
Comment (MikeMcQuaid) [inline topics/maven/index.md]: Does this need the quotes?

Comment (RohanNagar) [reply topics/maven/index.md]: Good catch on these, it shouldn't need the quotes. Updated!

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @RohanNagar

**Pull request merged**